### DISCUSSION TITLE
View and checkout merge requests

### DISCUSSION THREAD

**Discussion by AndiLeni** [gitextensions/gitextensions#10029] [Q&A]
Hello,

I have a GitLab repo from my university, where students solve tasks and hand them in as a merge request.
Is it somehow possible to view and checkout those merge requests in gitextensions?

Kind regards
Andreas

**Comment by mstv** [comment]
For pull requests on Github, I use a script added to the main toolbar. ("upstream" is to be replaced with the name of the repo to fetch from.)

![image](https://user-images.githubusercontent.com/36601201/172449894-f7a84464-0e61-40f3-8520-ffbb2076d79d.png)

**Comment by pmiossec** [comment]
Other possibility add a config line so that all gitlab merge requests are fetched when you are doing a normal 'git fetch':
https://stackoverflow.com/a/55387516/717372
